modules/cnv-remove-boot-order-item-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-remove-boot-order-item-web_{context}"]
+
+= Removing items from a boot order list in the  web console
+
+Remove items from the boot order list.
```

Comment:
I think this should be `Remove items from a boot order list by using the web console.`

Reply:
Please see my earlier question - do we want to switch over the heading text from gerunds (
Adding, Editing, Removing) to verbs (Add, Edit, Remove)...

Reply:
Lavanya, for these two I'm referring to the intro sentence, not the title.